src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SymbolCompletionProviderTests.cs

```diff
@@ -10733,5 +10733,23 @@ static void M()
     }
 }", "Local");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        [WorkItem(1152109, "https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1152109")]
+        public async Task NoItemWithEmptyDisplayName()
+        {
+            var markup = @"
+class C
+{
+    static void M()
+    {
+        int$$
```

Comment:
Shouldn't a space exist between `int` and `$$`?

```suggestion
        int $$
```

Reply:
Adding a space would change the context for completion, in which case no completion will trigger. Basically, what the current test does is to list all items at same context as `int` (which is treated as a local declaration and a local symbol with no name and of type int is generated, i.e. the one we need to filter out)